tests/test_postal_address.py

```diff
@@ -239,6 +239,14 @@ def test_postcode(address, expected_postcode):
         ''',
         True,
     ),
+    (
+            r'''
+            >123 Example Straße
+            SW1A 1AA
+            Deutschland
+            ''',
+            True,
```

Comment:
Can you make the indentation match the other examples pls? Sorry for the pedantry…
```suggestion
        r'''
        >123 Example Straße
        SW1A 1AA
        Deutschland
        ''',
        True,
```